src/pages/FarmV3/FarmTableComponents/FarmTableBalanceItem.tsx

```diff
@@ -542,8 +544,8 @@ const CollapsibleContent: FC<{
 
   const depositWithdrawOnClick = (): void => openActionModal(modeOfOperation.toLowerCase())
   const canDeposit = userTokenBalance >= MIN_AMOUNT_DEPOSIT
+  //userDepositedAmount?.lte(new BN(withdrawAmount)
```

Comment:
I got What this comment means, but cannot get why it is here. Code is quite clear.